src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Button, EntityHeader, Flex, Sans } from "@artsy/palette"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { TouchableHighlight, TouchableWithoutFeedback } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface FeaturedArtistsProps {
+  featuredArtists: any
+}
+
+interface FeaturedArtistsState {
+  showMore: boolean
+}
+
+export class FeaturedArtists extends React.Component<FeaturedArtistsProps, FeaturedArtistsState> {
+  state = {
+    showMore: false,
+  }
+
+  handleTap = (context: any, href: string) => {
+    SwitchBoard.presentNavigationViewController(context, href)
+  }
+
+  getFeaturedArtistEntityCollection = (artists: any) => {
+    return artists.map((artist, index) => {
+      const hasArtistMetaData = artist.nationality && artist.birthday
+      return (
+        <Box width="100%" key={index} pb={20}>
+          <TouchableWithoutFeedback onPress={() => this.handleTap(this, `/artist/${artist.slug}`)}>
```

Comment:
No need to create the `handleTap` method when the Switchboard call can be done inline here: ` onPress={() => SwitchBoard.presentNavigationViewController(this, `/artist/${artist.slug}`)}`